Assignment2.txt

```diff
@@ -0,0 +1,13 @@
+Name: Anya Lauria 
+
+1. If the operators are +-/*, then yes, they can be. 
```

Comment:
This is true - 2+ operators can be combined into a single line of code as long as each operator has the required amount of operands